.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
let's include .idea folder in .gitignore since it may not play well with gitbook and it can create confusion.

Reply:
sure, I will add these to .gitignore and commit it